src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -195,7 +200,8 @@ protected String buildItemKey(ConcurStandardAccountingExtractDetailLine saeLine)
         }
         
         return new StringBuilder(INITIAL_BUILDER_SIZE)
-                .append(makeEmptyIfBlank(saeLine.getChartOfAccountsCode()))
+                .append(makeEmptyIfBlank(saeLine.getReportId()))

```

Comment:
If report ID is blank, is that a line item error and should be rejected.  It is for PDP processing, not sure about collector.